planscore/website/templates/upload.html

```diff
@@ -44,109 +46,124 @@
+                <input type="hidden" name="success_action_redirect">
+                <input type="hidden" name="acl">
+                <input type="hidden" name="x-amz-security-token">
+                <p style="margin-bottom: 9px;">
+                    Upload a district plan as a geospatial file containing Polygon
+                    or MultiPolygon shapes. These file types are currently supported:
+                </p>
+                <ul>
+                    <li>
+                        <a href="https://doc.arcgis.com/en/arcgis-online/reference/shapefiles.htm">Shapefile</a>
+                        (upload as a <a href="https://en.wikipedia.org/wiki/Zip_(file_format)">single .zip file</a>)
+                    </li>
+                    <li><a href="http://geojson.org">GeoJSON</a></li>
+                    <li><a href="http://www.geopackage.org">Geopackage</a></li>
+                </ul>
+                <p>
+                    <input type="file" name="file" id="file">
+                    <label for="file" class="btn btn-primary">Select a file to upload</label>
+                </p>
+            </form>
+            <p>Processing time depends on the
                 complexity of the district plan, and may take a few seconds or
                 many minutes.
             </p>
-            <p>
-                <input type="file" name="file" id="file">
-                <label for="file" class="btn btn-default">1) Choose A File</label>
-                <input type="submit" name="submission" value="2) Score This Plan" class="btn btn-primary" disabled>
-            </p>
-            <p>
-                These file types and states are currently supported:
-            </p>
-            <ul>
-                <li>
-                    <a href="https://doc.arcgis.com/en/arcgis-online/reference/shapefiles.htm">Shapefile</a>
-                    (upload as a <a href="https://en.wikipedia.org/wiki/Zip_(file_format)">single .zip file</a>)
-                </li>
-                <li><a href="http://geojson.org">GeoJSON</a></li>
-                <li><a href="http://www.geopackage.org">Geopackage</a></li>
-            </ul>
-            <p style="margin-top:1em">
-                <img src="{{ digested_static_url('supported-states.svg') }}" alt="AK, AL, AR, AZ, CA, CO, DE, FL, GA, HI, IA, ID, IL, IN, KS, KY, LA, MA, MD, ME, MI, MN, MO, MT, NC, ND, NE, NH, NJ, NM, NV, OH, OK, OR, PA, RI, SC, SD, TN, TX, UT, VA, VT, WA, WI, WY">
-            </p>
-        </form>
+            
+            
+        </div>
     </section>
-	<script language="javascript">
+	<script>
 	
 	    var upload_form = document.getElementById('upload'),
 	        file_input = document.getElementById('file'),
-	        file_label = file_input.nextElementSibling;
+	        file_label = file_input.nextElementSibling,
+            file_label_text = file_label.textContent;
 	    
 	    enable_form('{{upload_fields_url}}', upload_form);
```

Comment:
This will need the full URL to work in prod since the dynamic parts come from api.planscore.org